src/main/java/net/pms/platform/windows/WindowsUtils.java

```diff
@@ -666,4 +666,12 @@ public static boolean uninstallWin32Service() {
 		return true;
 	}
 
+	/**
+	 * A readable way to see if this is Windows 11 or greater,
+	 * becuase Windows 11 identifies itself as 10.0.0, FFS...
```

Comment:
@SubJunk I really don't understand what this JavaDoc means - I mean "readable"? It seems to me like it still applies to Windows 10 as well, so shouldn't it at least say something about needing "improvement" to actually tell if it's Win11? I mean, there must be some way to tell using some Windows API.

Reply:
Good point, I'll use a name-based method instead

Reply:
There is another problem with this code - it seems that Windows is now never allowed to sleep in Windows 11, so the `doAllowSleep()` with the 5 minute delay isn't working. I can see that the code does run after 5 minutes in the logs.
Maybe it's because of the `run()` loop in `AbstractSleepWorker` conflicting with the timing, any ideas?